tests/amd_migraphx_test/runMIGraphXTests/runMIGraphXTests.cpp

```diff
@@ -233,523 +221,453 @@ int main(int argc, char **argv) {
+                        vx_size w = fread(buf, sizeof(float), input_dims_data_224x224[3], fp);
+                        if(w != input_dims_data_224x224[3]) {
+                            std::cerr << "ERROR: expected char[" << count*sizeof(float) << "], but got less in " << inputTensor_filename << std::endl;
+                            return -1;
+                        }
+                        for(size_t x = 0; x < input_dims_data_224x224[3]; x++) {
+                            *(buf+x) = *(buf+x) * mulVec[c] + addVec[c];
+                        }
+                    }
+                }
+            }
+            fclose(fp);
 
-        //squeezenet timing for 1000 iterations
-        t0 = clockCounter();
-        for(int i = 0; i < N; i++) {
-            ERROR_CHECK_STATUS(vxProcessGraph(graph_squeezenet));
-        }
-        t1 = clockCounter();
-        float squeezenetTime = (float)(t1-t0)*1000.0f/(float)freq/(float)N;
-        printf("OK: squeezenet took %.3f msec (average over %d iterations)\n", squeezenetTime, N);
-
-        //squeezenet results
-        status = vxMapTensorPatch(output_tensor_squeezenet, output_num_of_dims_4, nullptr, nullptr, &map_id, stride,
-            (void **)&ptr, VX_READ_ONLY, VX_MEMORY_TYPE_HOST);
-        if (status) {
-            std::cerr << "ERROR: vxMapTensorPatch() failed for output tensor" << std::endl;
-            return status;
-        }
+            status = vxUnmapTensorPatch(input_tensor_224x224, map_id);
+            if (status) {
+                std::cerr << "ERROR: vxUnmapTensorPatch() failed for imagenet" << status << ")" << std::endl;
+                return status;
+            }
+            if (runResnet50) {
+                //output tensor
+                vx_tensor output_tensor_resnet50 = vxCreateTensor(context, output_num_of_dims_2, output_dims_data_1x1000, VX_TYPE_FLOAT32, 0);
+                
+                //graph creation
+                vx_graph graph_resnet50 = vxCreateGraph(context);
+                status = vxGetStatus((vx_reference)graph_resnet50);
+                if(status) {
+                    printf("ERROR: vxCreateGraph(...) for renset50 failed (%d)\n", status);
+                    return -1;
+                }
+
+                vx_node node_resnet50 = amdMIGraphXnode(graph_resnet50, binaryFilename_resnet50_str.c_str(), input_tensor_224x224, output_tensor_resnet50);
+                ERROR_CHECK_OBJECT(node_resnet50);
+                ERROR_CHECK_STATUS(vxVerifyGraph(graph_resnet50));
+                ERROR_CHECK_STATUS(vxProcessGraph(graph_resnet50));
+
+                //resnet50 timing for 1000 iterations
+                t0 = clockCounter();
+                for(int i = 0; i < N; i++) {
+                    ERROR_CHECK_STATUS(vxProcessGraph(graph_resnet50));
+                }
+                t1 = clockCounter();
+                float resnet50Time = (float)(t1-t0) * 1000.0f / (float)freq / (float)N;
```

Comment:
add a space before and after -. there are multiple occurrences of t1-t0 in this file. please fix the formatting for all of them. Thanks.